chart/values.yaml

```diff
@@ -87,35 +87,9 @@ promscale:
   # If you are providing your own secret name, do
   # not forget to configure at below connectionSecretName
 
-  # selector used to provision your own Secret containing connection details
-  # Use this option with caution
-
   # if you are adding a conn string here do not forget
   # to add the same for kube-prometheus.grafana.timescale.adminPassSecret
-  connectionSecretName: ""
-
-  ## Note:
-
-  # If you using tobs deploy TimescaleDB do not configure below
-  # any connection details below as tobs will take care of it.
-
-  # connection details to connect to a target db
-  connection:
-    # Database connection settings. If `uri` is not
-    # set then the specific user, pass, host, port and
-    # sslMode properties are used.
-    uri: ""
```

Comment:
What needs to happen now if someone wants to connect via uri?

Reply:
They should do the following:
```
promscale:
  connectionSecretName: ""
  connection:
    uri: XYZ
```

This would disable the logic from this PR (which should be disabled either way) and allow connecting promscale to some, potentially external, database.